### PR TITLE
Release: Makefiles: Alphabetize and wrap

### PR DESCRIPTION
**Pull request by ocelotsloth**
* **What does this PR do?**

Alphabetizes and wraps lists in the files listed in issue #1835.

* **Does this PR meet the acceptance criteria?** (This is just a reminder for you,
  this section can be removed if you fulfill it.)

   - Documentation created/updated (you have to edit
     [doc/manual.xml.head](https://www.github.com/neomutt/neomutt/blob/master/doc/manual.xml.head)
     for that)

   - All builds and tests are passing

   - Added [doxygen code documentation](https://neomutt.org/dev/doxygen)
     [syntax](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html)

   - Code follows the [style guide](https://neomutt.org/dev/coding-style)

### CODE REVIEW

File: auto.def [hunk 1]
```diff
@@ -114,9 +114,10 @@ options {
 if {1} {
   # Keep sorted, please.
   foreach opt {
-    autocrypt bdb backtrace coverage doc everything fmemopen full-doc gdbm gnutls gpgme
-    gss homespool idn idn2 inotify kyotocabinet lmdb locales-fix lua mixmaster
-    nls notmuch pkgconf pgp qdbm sasl smime sqlite ssl testing tokyocabinet
+    autocrypt backtrace bdb coverage doc everything fmemopen full-doc gdbm
+    gnutls gpgme gss homespool idn idn2 inotify kyotocabinet lmdb locales-fix
+    lua mixmaster nls notmuch pgp pkgconf qdbm sasl smime sqlite ssl testing
+    tokyocabine
```
Comment (flatcap) [inline auto.def]:
```suggestion
    tokyocabinet
```

Comment (ocelotsloth) [reply auto.def]: Ah thanks for finding that. I just fixed it.

**Comment by ocelotsloth** [comment]
I left the lists in test/makefile unwrapped because it seemed like it would harm the readability. I'll go ahead and wrap them if you disagree.

**Comment by flatcap** [comment]
> I left the lists in test/makefile unwrapped because it seemed like it would harm the readability

Yeah, fair enough.

File: contrib/Makefile.autosetup [hunk 2]
```diff
@@ -1,7 +1,7 @@
-SAMPLES=	colors.default colors.linux gpg.rc Mush.rc Pine.rc \
-		sample.mailcap sample.neomuttrc sample.neomuttrc-starter sample.neomuttrc-tlr smime.rc \
-		smime_keys_test.pl Tin.rc mairix_filter.pl
-
+SAMPLES=	colors.default colors.linux gpg.rc mairix_filter.p Mush.rc Pine.rc \
+		sample.mailcap sample.neomuttrc sample.neomuttrc-starter \
+		sample.neomuttrc-tlr smime.rc smime_keys_test.pl Tin.rc 
+
```
Comment (flatcap) [inline contrib/Makefile.autosetup]: A couple of trailing space chars, here.

Comment (ocelotsloth) [reply contrib/Makefile.autosetup]: Just removed them:

https://github.com/neomutt/neomutt/compare/849c16d6f8917a37bb237a9ab9e22f3f45c17d90..deb302a74aeb0ddb03ca13a3ce4ef6c54d4eec28

**Pull request merged**